### PR TITLE
Frontend Assessment Improvement

### PR DESCRIPTION
**Pull request by itumoraes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
+
+You don't need to develop the best layout in this phase. Feel free to create your visual representation of Pipefy or replicate the visual structure that we already have.
+
+### Visual Representation
+![image](https://user-images.githubusercontent.com/5097397/127360349-7231b194-0a8c-4c31-af6c-886005bc8d6c.png)
+
+**OBS: You don't need to develop the favorite and "add new pipe" interactions. Don't worry about the Pipe's icons.**
+
+## What we will evaluate
+- Good and semantic HTML;
```
Comment (hermesnetto) [inline README.md:14]: Good what? haha

File: README.md [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## API
+You can access our API documentation with the below link:
 
-## Available Scripts
+[Pipefy's GraphQL Playground](https://app.pipefy.com/graphiql)
 
-In the project directory, you can run:
+## Tips
+1. You can use use the `organization` query to load all pipes. The other (cards) query is very simple to find in our documentation;
```
Comment (hermesnetto) [inline README.md:44]:
```suggestion
1. You can use the `organization` query to load all pipes. The other (cards) query is very simple to find in our documentation;
```

File: README.md (hunk 1 above)
Comment (itumoraes) [reply README.md:14]: A Good HTML :smile:

Comment (itumoraes) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (itumoraes) [reply README.md]: Done

**Review by hermesnetto — APPROVED** [review]

**Pull request merged**